data/json/martialarts.json

```diff
@@ -933,7 +934,7 @@
             {
                 "id" : "venom_snake_combo_initiate",
                 "name" : "Viper Ambush",
-                "description" : "You've lured 'em in!  Your next attack will be a Viper Bite.",
+                "description" : "You've lured 'em in! Your next attack will be a Viper Bite.",
```

Comment:
This is both incorrect and unrelated change and it shouldn't be done in current PR. We're using double whitespaces after end of each sentence since we're using monospace fonts.